packages/labs/test/multiSelectTests.tsx

```diff
@@ -44,6 +44,11 @@ describe("<MultiSelect>", () => {
         assert.equal((input.getDOMNode() as HTMLInputElement).placeholder, placeholder);
     });
 
+    it("tags can be bolded", () => {
```

Comment:
change title to accurately describe goal of this test: "tagRenderer can return JSX"

(`<b>` is the "old bold," `<strong>` is the "new bold": https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#Bold_vs._Strong. we don't actually care that tags can be bolded, but that they can contain JSX.)